tests/Nfield.Quota.Tests/QuotaFrameValidatorTests.cs

```diff
@@ -1021,5 +1023,106 @@ public void ComplexFrame_VariablesShouldHaveAtLeastOneVisibleLevel()
                 result.Errors.Single().ErrorMessage,
                 Is.EqualTo("Quota frame invalid. Frame has variables with no visible levels. Affected variable name: 'varName2'. If you don't care about any levels under variable 'varName2', consider hiding that variable instead."));
         }
+
+        [Test]
+        public void Definitions_OdinVariableShouldBePresent()
+        {
+            var var1Id = Guid.NewGuid();
+            var levelId = Guid.NewGuid();
+
+            var levelDefinitions = new[]
+            {
+                new QuotaLevelDefinition
+                {
+                    Id = levelId,
+                    Name = "Level 1"
+                }
+            };
+
+            var definitions = new[]
+            {
+                new QuotaVariableDefinition(levelDefinitions)
+                {
+                    Id = var1Id,
+                    Name = "var 1"
+                    // OdinVariableName = 
+                }
```

Comment:
I would like it if we test for 'null', because that is what you get when a variable is omitted.
OdinVariableName = null
and rename the test to 'Definitions_OdinVariableCannotBeNull'